src/components/CachedImage/CachedImage.tsx

```diff
@@ -41,7 +41,15 @@ class CachedImage extends React.Component<CachedImageProps, CachedImageState> {
     }
   }
 
+  addImagesToCache(){
+    if("caches" in window) {
```

Comment:
We use [prettier](https://prettier.io/) for code formatting. Could you apply it to your changes?

Reply:
@maarcingebala Actually I just wanted to ask about it. I don't see any configs or prettier commands in the project. Are you using default?

Reply:
Yep, I already spoke with @piotrgrundas that `prettier` should come with the project in `package.json`. Also, we could add it to CI to automatically fail PRs that introduce unformatted code. We're using the default config.

Reply:
ATM just use prettier whit default settings and auto formatting on save.